tests/unit/test_reward_aggregation.py

```diff
@@ -53,6 +53,7 @@ def test_aggregate_orderbook_rewards(self):
             "0x007",
         ]
         amounts = [39, 0, 40, 0, 41, 50, 60, 70, 40, 50, 0]
+        surplus_fees = [None] * len(amounts)
```

Comment:
I am glad you noticed this. However in production, this kind of trick should **NEVER** be used because that same instance of None is allocated to every slot of the array. If you do array[0] = 5

Then the entire array will be filled with 5. Well thats how it works in older versions of python... I guess its not a thing anymore....

<img width="185" alt="Screenshot 2022-12-08 at 14 13 14" src="https://user-images.githubusercontent.com/11778116/206455430-557ecc4b-172a-4a22-b2e0-683b0ea5afe5.png">